modules/dsp/chowdsp_math/Math/chowdsp_BufferMath.cpp

```diff
@@ -93,6 +93,12 @@ static void copyBufferData (const BufferType1& bufferSrc, BufferType2& bufferDes
         const auto* srcData = bufferSrc.getReadPointer (ch);
         auto* destData = bufferDest.getWritePointer (ch);
 
+        jassert(destData != nullptr);
+        jassert(srcData != nullptr);
+        if (!destData || !srcData) {
+            continue;
+        }
```

Comment:
Under what conditions would these assertions be hit? Since `buffer.getReadPointer()` doesn't check bounds, it often returns a garbage pointer rather than a `nullptr` when called with an invalid argument.

Reply:
Not super sure they are as releveant now, but these were being hit without the other change, not sure if they are as strictly neccecary now, but could have caught the above before far as I understand.

Reply:
For sure... in that case, could we keep the assertions, but remove the if statement?

Reply:
Done! :)